cfme/tests/services/test_pxe_service_catalogs.py

```diff
@@ -67,8 +67,6 @@ def setup_pxe_servers_vm_prov(pxe_server, pxe_cust_template, provisioning):
     if not pxe_server.exists():
         pxe_server.create()
     pxe_server.set_pxe_image_type(provisioning['pxe_image'], provisioning['pxe_image_type'])
-    if not pxe_cust_template.exists():
```

Comment:
**REQUIRED**: Why do we no longer assert that the template is there?

Reply:
@psav - calling pxe_cust_template - executes 'get_template_from_config' which  already creates an UI object - no need to check it once again